### PR TITLE
Added guidelines.

### PR DESCRIPTION
**Pull request by SharviE29**
License and references are not added yet. If there are any additions/deletions that are to be made in this code,  I'll make sure I'll make them. Thank you.

### CODE REVIEW

**Comment by iamAbhishekkumar** [comment]
I think references will be added after we complete the project. :)

**Comment by SharviE29** [comment]
> I think references will be added after we complete the project. :)

Ok , so is there anything that has to be added in this or is this okay?

**Comment by iamAbhishekkumar** [comment]
I think, for contribution guidelines this is fine.

**Comment by SharviE29** [comment]
> I think, for contribution guidelines this is fine. Project admin, will soon accept your pr :)

Ok , Thanks!

File: README.md [hunk 1]
```diff
@@ -7,3 +7,58 @@ Imagine the PokeDex Ash used to carry around in his pocket 🤔
...
+
+3. Build app
+        
+        flutter run
+
+4. Create a new branch from master/main branch
+
+        git checkout -b add "your branch name"
+
+5. Make the changes in the code
```
Comment (infiniteoverflow) [inline README.md:38]: In order to commit the changes, you need to "add" the changes. 
You missed the code `git add <list-of-files-changed>`

And provide a strict warning to add only those folder which have been modified. Avoid using `git add .`

**Review by infiniteoverflow — CHANGES_REQUESTED** [review]

**Comment by infiniteoverflow** [comment]
@SharviE29 you are not supposed to make a PR without creating an issue, and also before creating an issue always reach out to the admin or mentors.

Feel free to reach out to me on the discord server if you have any queries.

@SharviE29 fix the changes that I have requested, then I will merge this PR

**Comment by SharviE29** [comment]
> @SharviE29 you are not supposed to make a PR without creating an issue, and also before creating an issue always reach out to the admin or mentors.
> 
> Feel free to reach out to me on the discord server if you have any queries.
> 
> @SharviE29 fix the changes that I have requested, then I will merge this PR

Actually sir there was already an issue that was created for this, so I didn't create one. But yes, I'll keep this in mind. I'll make the changes right away. Thank you.

**Pull request closed without merging**